addons/advanced_fatigue/ACE_Settings.hpp

```diff
@@ -20,26 +20,30 @@ class ACE_Settings {
         description = CSTRING(PerformanceFactor_Description);
         typeName = "SCALAR";
         value = 1;
+        sliderSettings[] = {0, 10, 1, 1};
```

Comment:
Should be `sliderSettings[] = {0, 3, 0.1, 1};` which translates to `GVAR(VO2Max) >= 35 && GVAR(VO2Max) <= 95`.

Reply:
I have it at 5 max, just to keep it consistent with the other settings in the module.

Reply:
Would have to ask @BaerMitUmlaut what he thinks about it.